bin/dpipe.ml

```diff
@@ -75,7 +78,7 @@ let nat a b =
     try
       let v = int_of_string s in
       let p = Printf.sprintf in
-      match v, a, b with
+      match (v, a, b) with
```

Comment:
Oh, I see now that the GZIP patch was rebased on this PR, should put my pedantic questions about extraneous parens here instead. :)
Why do we need to have parens here?